alpine/license.md

```diff
@@ -0,0 +1 @@
+View [license information](https://github.com/gliderlabs/docker-alpine/blob/master/LICENSE) for the software contained in this image.
```

Comment:
You are correct, that license was for the build script we made (someone actually requested the licensing terms for it so we added one). Looking in to it more, I can't actually find a license for Alpine Linux. Seems strange that there wouldn't be one. I'll dig a little more.